index.html

```diff
@@ -10892,6 +10892,44 @@ <h2>A project for first-time contributions</h2>
                     </div>
                 </div>
                 <!-- ________ Joseph's card END ________  -->
+		    
+		    <!-- ________ Kamva's card START ________  -->
+                <div class="card">
```

Comment:
You need to paste your card exactly above the line (which would be towards the end of this file)
```
<!-- ^^^^ Paste YOUR CARD directly ^^ ABOVE ^^ this line ^^^^ -->
```

Reply:
Thank you for taking your to help! I will do so